components/add_groups_to_channel_modal/add_groups_to_channel_modal.test.jsx

```diff
@@ -75,6 +77,7 @@ describe('components/AddGroupsToChannelModal', () => {
         wrapper.setState({values: []});
         await wrapper.instance().handleSubmit({preventDefault: jest.fn()});
         expect(actions.linkGroupSyncable).not.toBeCalled();
+        expect(instance.handleHide).not.toBeCalled();
```

Comment:
Also  add the following wouldn't hurt
```suggestion
        expect(instance.handleResponse).not.toBeCalled();
        expect(instance.handleHide).not.toBeCalled();
```